src/js/jsx/sections/libraries/LibraryBar.jsx

```diff
@@ -117,6 +119,38 @@ define(function (require, exports, module) {
                     <Gutter />
                 </div>
             );
+        },
+        
+        _createColorButton: function (attr, buttonText) {
+            var layer = this.props.selectedLayers.first();
+            if (this.props.selectedLayers.size !== 1 || layer[attr].isEmpty()) {
```

Comment:
`What if all the selected layers have the same color?` 
good question. In standard PS, you can only add fill / stroke / character / effect from single layer, so I followed the same behavior.